lib/util/git/private-key.spec.ts

```diff
@@ -1,5 +1,7 @@
-import { mocked } from '../../../test/util';
-import * as exec_ from '../exec';
+import fs from 'fs-extra';
+import { MockProxy, any, mockReset } from 'jest-mock-extended';
+import { Fixtures } from '../../../test/fixtures';
+import * as execModule from '../exec';
```

Comment:
please don't rename for no benefit

Reply:
Reverted: b2e11ca77d84226013bf28aaa097817df8f6c12f